### PR TITLE
refactor: move all models to their own unique containers

### PR DESCRIPTION
**Pull request by sambarnes**
## Details

pause dynamic model addition development & simplify model:container mapping to be 1-to-1

this allows for easier cost tracking by model, directly inside of modal instead of datadog

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/OpenRouterTeam/openrouter-runner/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I agree to license this contribution under the MIT LICENSE
- [x] I checked the [current PR](https://github.com/OpenRouterTeam/openrouter-runner/pulls) for duplication.

### CODE REVIEW

File: modal/runner/containers/__init__.py [hunk 1]
```diff
@@ -1,47 +1,17 @@
...
-    VllmContainerA100_40G,
-    VllmContainerA100_80G,
-    VllmContainerA100_160G,
-    VllmContainerA100_160G_Isolated,
+    VllmContainer_IntelNeuralChat7B,
+    VllmContainer_JebCarterPsyfighter13B,
+    VllmContainer_JohnDurbinBagel34B,
+    VllmContainer_KoboldAIPsyfighter2,
+    VllmContainer_MicrosoftPhi2,
+    VllmContainer_NeverSleepNoromaidMixtral8x7B,
```
Comment (sambarnes) [inline modal/runner/containers/__init__.py:7]: these names are admittedly meh, but probs doesnt matter much. open to changing

File: modal/shared/protocol.py [hunk 2]
```diff
@@ -77,17 +46,12 @@ class Params(BaseModel):
...
-    container: ContainerType
-
-
 class CompletionPayload(BaseModel):
     id: str
     prompt: str
     stream: bool = False
     params: Params
     model: str
-    runner: RunnerConfiguration | None = None
```
Comment (sambarnes) [inline modal/shared/protocol.py:90]: opted to get rid of this complexity so theres less going on.  just a simple 1-to-1 mapping for model->container

File: scripts/shared.ts [hunk 3]
```diff
@@ -54,10 +52,6 @@ export async function completion(
     stream
   };
 
-  if (container) {
-    bodyPayload['runner'] = { container };
-  }
-
```
Comment (sambarnes) [inline scripts/shared.ts:60]: need to check if -web repo populates this field, but removed everywhere in this repo

File: modal/shared/protocol.py (hunk 2 above)
Comment (sambarnes) [reply modal/shared/protocol.py:90]: looks like -web repo does not use this field so we good

https://github.com/OpenRouterTeam/openrouter-web/blob/05061d4f8bb3ae41b207a6784c78c9da7401a05c/utils/adapters/modal.ts#L81-L101

**Review by alexanderatallah — APPROVED** [review]

**Review by louisgv — APPROVED** [review]
lgtm 👍

**Pull request merged**